testing/fzf/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Kevin Daudt <ops@ikke.info>
+# Maintainer: Kevin Daudt <ops@ikke.info>
+pkgname=fzf
+_pkgver="0.17.0-2"
+pkgver=${_pkgver/-/.}
+pkgrel=0
+pkgdesc="A command-line fuzzy finder"
+url="https://github.com/junegunn/fzf"
+arch="all"
+license="MIT"
+depends="go"
+subpackages="$pkgname-tmux"
+source="$pkgname-$_pkgver.tar.gz::https://github.com/junegunn/fzf/archive/$_pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$_pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+check() {
+	cd "$builddir"
+	make test
+}
+
+package() {
+	cd "$builddir"
+
+	# Just copies the target binary to $buildir/bin
+	make install
+
+	install -Dm0755 bin/fzf $pkgdir/usr/bin/fzf
+}
+
+tmux() {
+	depends="tmux bash"
```

Comment:
we need to ad tmux and bash to makedepends as a workaround to help builder figure out the build order properly. (image if something else depends on fzf-tmux before tmux or bash is built)

Reply:
I've added a `makedepends="$depends"`